### ISSUE TITLE
Why assign class splits specifically in code ? 

### ISSUE THREAD

**Issue by akira-l** [iCGY96/ARPL#1]
I noticed that the classes of close set are assigned in split.py. Are the results only reimplemented via the specific classes ? Two phenomenons shown in experiments. 
1. I tried to change the class number into random set. The performance decrease dramatically. The average AUROC in 5 times is only about 50. 
2. I trained the close set with close set with cross entropy loss, and use the softmax probability as logits in evaluation. Then, I get similar perfromance (about 74~ in AUROC) in results. May I miss some something in your code ?

**Comment by iCGY96** [comment]
Our experiment is based on the provided split.py for a fair comparison, but other data split should not make such a big difference. I'm not sure if you set some parameters accurately. You can send the relevant data and code to my email (gy.chen@pku.edu.cn ）, and I'll try it for you.
